src/DockerDiskUsage.php

```diff
@@ -0,0 +1,536 @@
+<?php
+
+namespace BiffBangPow\DockerDiskUsage;
+
+use Colors\Color;
+use Docker\Docker;
+use Stringy\Stringy;
+
+class DockerDiskUsage
+{
+    const BYTES_TO_GIGABYTES = 1024 * 1024 * 1024;
+
+    const DETAILED_REPORT = true;
+    const SUMMARY_REPORT = false;
+
+    const FORMATTED_TEXT = true;
+    const PLAIN_TEXT = false;
+
+    const INCLUDE_MOUNTS = true;
+    const EXCLUDE_MOUNTS = false;
+
+    /**
+     * @var Color
+     */
+    private $cliFormatter;
+
+    /**
+     * @var array
+     */
+    private $data = [];
+
+    /**
+     * @var Docker
+     */
+    private $docker;
+
+    /**
+     * @var array
+     */
+    private $errors = [];
+
+    /**
+     * @var array
+     */
+    private $images = [];
+
+    /**
+     * @var bool
+     */
+    private $includeMounts = self::EXCLUDE_MOUNTS;
+
+    /**
+     * @var bool
+     */
+    private $useTextFormatting = self::FORMATTED_TEXT;
+
+    /**
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->cliFormatter = new Color();
+        $this->docker = new Docker();
```

Comment:
Getting `Class 'Docker\Docker' not found in /home/james/Sites/docker-disk-usage/src/DockerDiskUsage.php:63`